app/controllers/v0/mhv_opt_in_flags_controller.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module V0
+  class MHVOptInFlagsController < ApplicationController
+    def show
+      opt_in_flag = MHVOptInFlag.find_by(user_account_id: current_user.user_account, feature: params[:feature])
+      raise Common::Exceptions::RecordNotFound if opt_in_flag.nil?
```

Comment:
Weirdly, raising this particular error looks like it expects a message argument, otherwise it actually sends back a different error to the caller (here's the error I get when I call it: `{"errors":"wrong number of arguments (given 0, expected 1)"}`

What I'd do is give this a simple message, (like just 'Record Not Found'), and then in the rescue, render the message instead of the error:
`render json: { errors: e.message }, status: :not_found`